Although, I see another issue with running `corepack pnpm i` https://github.com/enso-org/enso/actions/runs/13249248686/job/37024762090?pr=12252#step:10:1788

```
INFO ide_ci::program::command: sbt ℹ️ [info] . postinstall: ERROR: C:/_bazel/external/rules_rust+/crate_universe/private/common_utils.bzl:55:13: Traceback (most recent call last):
 INFO ide_ci::program::command: sbt ℹ️ [info] . postinstall: 	File "C:/_bazel/external/rules_rust+/crate_universe/extensions.bzl", line 990, column 37, in _crate_impl
 INFO ide_ci::program::command: sbt ℹ️ [info] . postinstall: 		_generate_hub_and_spokes(
 INFO ide_ci::program::command: sbt ℹ️ [info] . postinstall: 	File "C:/_bazel/external/rules_rust+/crate_universe/extensions.bzl", line 626, column 51, in _generate_hub_and_spokes
 INFO ide_ci::program::command: sbt ℹ️ [info] . postinstall: 		splice_outputs = splice_workspace_manifest(
 INFO ide_ci::program::command: sbt ℹ️ [info] . postinstall: 	File "C:/_bazel/external/rules_rust+/crate_universe/private/splicing_utils.bzl", line 170, column 19, in splice_workspace_manifest
 INFO ide_ci::program::command: sbt ℹ️ [info] . postinstall: 		cargo_bazel_fn(
 INFO ide_ci::program::command: sbt ℹ️ [info] . postinstall: 	File "C:/_bazel/external/rules_rust+/crate_universe/private/common_utils.bzl", line 88, column 23, in _execute
 INFO ide_ci::program::command: sbt ℹ️ [info] . postinstall: 		return execute(
 INFO ide_ci::program::command: sbt ℹ️ [info] . postinstall: 	File "C:/_bazel/external/rules_rust+/crate_universe/private/common_utils.bzl", line 55, column 13, in execute
 INFO ide_ci::program::command: sbt ℹ️ [info] . postinstall: 		fail(_EXECUTE_ERROR_MESSAGE.format(
 INFO ide_ci::program::command: sbt ℹ️ [info] . postinstall: Error in fail: Command [C:/_bazel/modextwd/rules_rust++crate/cargo-bazel.exe, "splice", "--output-dir", C:/_bazel/modextwd/rules_rust++crate/crates/splicing-output, "--splicing-manifest", C:/_bazel/modextwd/rules_rust++crate/crates/splicing_manifest.json, "--config", C:/_bazel/modextwd/rules_rust++crate/crates/config.json, "--cargo-lockfile", C:/runner/_work/enso/enso/Cargo.lock, "--cargo", "C:/_bazel/external/rules_rust++rust_host_tools+rust_host_tools/bin/cargo.exe", "--rustc", "C:/_bazel/external/rules_rust++rust_host_tools+rust_host_tools/bin/rustc.exe"] failed with exit code 256.
```

I'll try to see if it is SBT-related or not

<blockquote><img src="https://opengraph.githubassets.com/dc40a18fedeb0f7f80ee9396be9ca34f61c3d4dec1055b0df42e7fa0497acade/enso-org/enso" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/enso-org/enso">Enable ydoc-server tests · enso-org/enso@d3e5f6b</a></strong></div><div>Enso Analytics is a self-service data prep and analysis platform designed for data teams. - Enable ydoc-server tests · enso-org/enso@d3e5f6b</div></blockquote>

--- a/project/Ydoc.scala
+++ b/project/Ydoc.scala
@@ -5,8 +5,11 @@ import scala.sys.process.*
 
 object Ydoc {
 
+  private val corepackCommand =
+    if (Platform.isWindows) "corepack.cmd" else "corepack"
+
   private val pnpmCommand =
-    if (Platform.isWindows) "corepack.cmd pnpm" else "corepack pnpm"
+    s"$corepackCommand pnpm"
 
   /** Generates the bundled JS source of the Ydoc server.
     *
@@ -53,10 +56,11 @@ object Ydoc {
           ydocServerResourceManaged / "org" / "enso" / "ydoc" / "server" / "ydoc.cjs"
 
         if (changed) {
-          val command = s"$pnpmCommand build:ydoc-server-polyglot"
-          streams.log.info(command)
-          val exitCode = command ! streams.log
+          runCommand(s"$corepackCommand --version", streams)
+          runCommand(s"$pnpmCommand --version", streams)
 
+          val command = s"$pnpmCommand build:ydoc-server-polyglot"
+          val exitCode = runCommand(command, streams)
           if (exitCode != 0) {
             throw new CommandFailed(command, exitCode)
           }
@@ -87,9 +91,11 @@ object Ydoc {
       case (changed, _) =>
         val nodeModules = base / "node_modules"
         if (changed || !nodeModules.isDirectory) {
+          runCommand(s"$corepackCommand --version", streams)
+          runCommand(s"$pnpmCommand --version", streams)
+
           val command = s"$pnpmCommand i --frozen-lockfile"
-          streams.log.info(command)
-          val exitCode = command ! streams.log
+          val exitCode = runCommand(command, streams)
           if (exitCode != 0) {
             throw new CommandFailed(command, exitCode)
           }
@@ -99,6 +105,17 @@ object Ydoc {
     val inputFile = base / "pnpm-lock.json"
 
     generator(inputFile)
+  }
+
+  /** Run command printing the output to the log stream.
+    *
+    * @param command the command to run
+    * @param streams the build streams
+    * @return exit code
+    */
+  private def runCommand(command: String, streams: TaskStreams): Int = {
+    streams.log.info(command)
+    command ! streams.log
   }
 
   final private class CommandFailed(command: String, exitCode: Int)

--- a/project/Ydoc.scala
+++ b/project/Ydoc.scala
@@ -59,7 +59,7 @@ object Ydoc {
           runCommand(s"$corepackCommand --version", streams)
           runCommand(s"$pnpmCommand --version", streams)
 
-          val command = s"$pnpmCommand build:ydoc-server-polyglot"
+          val command  = s"$pnpmCommand build:ydoc-server-polyglot"
           val exitCode = runCommand(command, streams)
           if (exitCode != 0) {
             throw new CommandFailed(command, exitCode)
@@ -94,7 +94,7 @@ object Ydoc {
           runCommand(s"$corepackCommand --version", streams)
           runCommand(s"$pnpmCommand --version", streams)
 
-          val command = s"$pnpmCommand i --frozen-lockfile"
+          val command  = s"$pnpmCommand i --frozen-lockfile"
           val exitCode = runCommand(command, streams)
           if (exitCode != 0) {
             throw new CommandFailed(command, exitCode)

